---
layout: page
title: Materials
permalink: /materials/
---

All materials used in this course will be online and freely available 
for re-use. We will also continue to provide the links for past materials. 

Topics are focused on the analysis of microbial metagenomic data including 
amplicon, shotgun metagenomics and the skills needed to use these tools, 
such as the command line and cloud computing. A more detailed overview is
below. 

We hope these materials can be helpful if you're teaching these topics yourself, or to use them for self-guided learning. If you do find them helpful, please [let us know](mailto:edamame.course@gmail.com)


## Materials by Year

<h3>2016 Tutorials</h3>
[Overview of 2016 materials](https://github.com/edamame-course/2016-tutorials/wiki)

<h3>2015 Tutorials</h3>
[Overview of 2015 materials](https://github.com/edamame-course/2015-tutorials/wiki)

<h3>2014 Tutorials</h3>
[Pages for 2014 tutorials](https://github.com/edamame-course/2014-tutorials/tree/master/_posts)

## 2016 Materials Organized by Learning Goals
* Computing literacy
   * [Intro to the Shell](https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-22-introduction_to_the_shell.md)
   * [tmux for remote sessions](https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-22_tmux.md)
   * [Getting started with GitHub](https://github.com/edamame-course/Github/blob/master/Tutorial.md)
   * [Computing workflows for biologists - paper](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002303)
* Cloud computing
    * [Amazon EC2 start-up](https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-22-EC2_Startup.md)
    * [File transfer to the EC2](https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-22-EC2_Connection_FileTransfer.md)
    * [Getting started with the EC2 - from Angus](http://angus.readthedocs.io/en/2015/amazon/index.html)
* Microbial amplicon analysis
    * [Assessing sequencing quality with FastQC (bonus: intro to automation, installing software on the EC2)](https://github.com/edamame-course/FastQC/blob/master/final/2016-06-22_FastQC_tutorial.md)
    * [Subsampling a large amplicon dataset for developing an analysis workflow](https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-23-QIIME1.md#ampliconsubsampling)
    * [Firing up the QIIME AMI](https://github.com/edamame-course/Amplicon_Analysis/blob/master/final/QIIME0.md)
    * [QIIME workflow overview](https://github.com/edamame-course/Amplicon_Analysis/blob/master/resources/QIIME_flow_chart_2016.pdf)
    * [QIIME tutorial](https://github.com/edamame-course/Amplicon_Analysis/blob/master/final/2016-07-13-QIIME1.md)
    * [mothur workflow](http://www.mothur.org/wiki/MiSeq_SOP)
* Microbial shotgun metagenome analysis 
    * [Examples of installing mg tools on the EC2](https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-25_MetaGenome_Installation.md)   
    * [Demo: using seqtk for subsampling a large metagenome dataset for developing an analysis workflow](https://github.com/edamame-course/2015-tutorials/blob/master/demos/Creating_SubSampled_mg_Data.sh)
    * [metaG sequencing preprocessing, quality control, and trimming](https://github.com/edamame-course/Metagenome/blob/master/2016-07-15-metaG-sequencing-preprocessing.md)
    * [Digital normalization](https://github.com/edamame-course/Metagenome/blob/master/2016-07-15-metaG-assembly.md)
    * [Assembly with MEGAHit](https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-25-assembling_your_metagenome.md)
    * [Evaluating assembly](https://github.com/edamame-course/Metagenome/blob/master/2016-07-15-assessing-metagenome-assembly.md)
    * [Estimating abundance from metagenomes](https://github.com/edamame-course/Metagenome/blob/master/2016-07-15-counting-abundance-with-mapped-reads.md)
    * [Binning assemblies](https://github.com/edamame-course/metagenome_binning/blob/master/binning_tutorial.md)
    * [Annotation of assembled reads](https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-26-MG_Annotation_Prokka.md)
    * [Xander for targeted gene assembly](https://github.com/edamame-course/Xander/blob/master/Xander.md)
* Ecological Statistics with R
    * [R basics](https://github.com/edamame-course/Jones_R_Files)
    * [Visualizations Demo](https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-07-01-VisualizationDemo.md)
    * [Quick intro to R for comparative (beta) diversity](https://github.com/edamame-course/2015-tutorials/blob/master/final/Ashley_Intro_to_R.md)
* Using Databases
    * [Local BLAST](https://github.com/edamame-course/BLAST-tutorial/blob/master/running-BLAST.md)
    * [Using APIs to access NCBI and MG-RAST data](http://adina-howe.readthedocs.org/en/latest/ncbi/)
    * [Getting data from NCBI](https://github.com/germs-lab/tutorial-blast-annotation/blob/master/ncbi/index.md)

***


